dcos_launch/sample_configs/gce-onprem.yaml

```diff
@@ -0,0 +1,21 @@
+---
+launch_config_version: 1
+deployment_name: dcos-onprem-gce
+installer_url: https://downloads.dcos.io/dcos/testing/master/dcos_generate_config.sh
+platform: gce
+provider: onprem
+os_name: coreos
+dcos_config:
+    cluster_name: My Awesome DC/OS
+    resolvers:
+        - 8.8.4.4
+        - 8.8.8.8
+    dns_search: mesos
+    master_discovery: static
+num_masters: 1
+num_private_agents: 2
+num_public_agents: 1
+key_helper: true
```

Comment:
Why would I want to remove this? I did want to remove the gce_template but not this